extension/jaegerremotesampling/README.md

```diff
@@ -13,5 +19,56 @@ extensions:
       endpoint: jaeger-collector:14250
   jaegerremotesampling/1:
     strategy_file: /etc/otel/sampling_strategies.json
+    grpc:
+      endpoint: :15251
+    http:
+      endpoint: :5878
```

Comment:
Looking at this config, kind of don't like, especially the "remote" vs "strategy_file". What about:

```yaml
  jaegerremotesampling:
    grpc:
      endpoint: :15251
    strategy:
      remote:
        endpoint: jaeger-collector:14250
  jaegerremotesampling/1:
    http:
      endpoint: :5878
    strategy:
       file: /etc/otel/sampling_strategies.json
```

So we separate the strategy config vs the other config.

Reply:
Looks better, but I would change it to `source` or `strategies`.  Strategy is ambiguous here, as it could mean the "retrieval strategy" or "sampling strategies". `source` is clearer, so, I'll change it to that, but let me know if you have another suggestion.

Reply:
PR updated.

Reply:
I don't see the update in the readme. Still using `strategy_file`

Reply:
Readme fixed